packages/react-native-editor/src/initial-html.js

```diff
@@ -7,6 +7,19 @@ export default `
 <h2 id="this-is-an-anchor">What is Gutenberg?</h2>
 <!-- /wp:heading -->
 
+
+<!-- wp:jetpack/story -->
+<div class="wp-block-jetpack-story wp-story"></div>
+<!-- /wp:jetpack/story -->
+
+<!-- wp:jetpack/story {"mediaFiles":[]}} -->
+<div class="wp-block-jetpack-story wp-story"></div>
+<!-- /wp:jetpack/story -->
+
+<!-- wp:jetpack/story {"mediaFiles":[{"alt":"","caption":"","id":320,"link":"https://cldup.com/n48GTQmw7s.jpg","mime":"image/jpeg","type":"image","url":"https://cldup.com/n48GTQmw7s.jpg"},{"alt":"","caption":"","id":321,"link":"https://cldup.com/u2kdydrlQv.jpg","mime":"image/jpeg","type":"image","url":"https://cldup.com/u2kdydrlQv.jpg"},{"alt":"","caption":"","id":319,"link":"https://cldup.com/fi7gqyUrbL.jpg","mime":"image/jpeg","type":"image","url":"https://cldup.com/fi7gqyUrbL.jpg"}]} -->
+<div class="wp-block-jetpack-story wp-story"></div>
+<!-- /wp:jetpack/story -->
+
```

Comment:
These could go bellow on the Jetpack section.

Reply:
👍 these were moved now in https://github.com/wordpress-mobile/gutenberg-mobile/pull/2611/commits/0f8916f5a4dccd263cfbecfb88e1a6df61cf09f2 after https://github.com/wordpress-mobile/gutenberg-mobile/pull/2690 was merged